try:
	pass
except ValueError:
	pass
except:
	pass
else:
	pass
finally:
	pass